intern/remote/boss/remote.py

```diff
@@ -906,26 +906,27 @@ def get_cutout(self, resource, resolution, x_range, y_range, z_range, time_range
 
     def get_experiment(self, coll_name, exp_name):
         """
-        Gets a json dictionary of the channel properties displayed on the boss
+        Convenience method that gets experiment resource.
         
         Args:
-            resource (intern.resource.Resource): A channel.
+            coll_name (str): Collection name
+            exp_name (str): Experiment name
 
         Returns:
-            (dict): Dictionary of channel properties
+            ExperimentResource
         """
         exp = ExperimentResource(exp_name, coll_name)
         return self.get_project(exp)
 
     def get_coordinate_frame(self, name):
         """
-        Gets the coordinate frame properties displayed under the coordinate frame on the boss
+        Convenience method that gets coordinate frame resource
 
         Args:
             name (str): Name of the coordinate frame
         
         Returns:
-            (dictionary): Dictionary of coordinate frame information.
+            CoordinateFrame Resource
```

Comment:
Enclose in parantheses for consistency with the rest of the docstrings and remove space.